lib/Sema/TypeCheckConstraints.cpp

```diff
@@ -2058,7 +2058,7 @@ bool TypeChecker::typeCheckCompletionSequence(Expr *&expr, DeclContext *DC) {
 
   // Attempt to solve the constraint system.
   SmallVector<Solution, 4> viable;
-  if (CS.solve(viable, FreeTypeVariableBinding::GenericParameters))
+  if (CS.solve(viable, FreeTypeVariableBinding::UnresolvedType))
```

Comment:
Looking into this a little bit, code completion calls ``typeCheckCompletionSequence`` for each potential operators to decide whether they should be included into the code completion list. It seems this line change caused a solution that used to be unviable for "+" now becomes viable, thus ``+`` is considered as a viable code completion  item. Any idea why this line change is less strict than before?